src/lib_ccx/ccx_decoders_isdb.c

```diff
@@ -1345,12 +1340,8 @@ int isdb_parse_data_group(void *codec_ctx,const uint8_t *buf, struct cc_subtitle
 		isdb_log("ISDB group B\n");
 	}
 
-	isdb_log("ISDB (Data group) version %d\n",version);
```

Comment:
Any specific reason for deleting this?

Reply:
I deleted version variable so CLion deleted it automatically , as I have restored it I'll add this line again.